assets/js/blocks/featured-product/block.js

```diff
@@ -161,12 +174,18 @@ const FeaturedProduct = ( {
 					<ToggleControl
 						label={ __( 'Show description', 'woo-gutenberg-products-block' ) }
 						checked={ attributes.showDesc }
-						onChange={ () => setAttributes( { showDesc: ! attributes.showDesc } ) }
+						onChange={ () =>
+							setAttributes( { showDesc: ! attributes.showDesc } )
+						}
 					/>
 					<ToggleControl
 						label={ __( 'Show price', 'woo-gutenberg-products-block' ) }
 						checked={ attributes.showPrice }
-						onChange={ () => setAttributes( { showPrice: ! attributes.showPrice } ) }
+						onChange={ () =>
+							setAttributes( {
```

Comment:
Same here, put the entire function declaration on the next line.

Reply:
Same as before. :point_up_2: